workshops/dodge/README.md

```diff
@@ -19,34 +19,65 @@ p5play is made by a really cool game company called [Molleindustria](http://www.
 
 ## Part I. Setup
 
-Though we're not using many HTML elements today, in order to display our game on a website, we need a web page, and for that we need an HTML file. Our HTML file will have several crucial elements, in addition to the standard base template you've seen before:
+Just as we've done previously, we're going to be building this project in Cloud9.
 
-```html
-<script src="http://cdnjs.cloudflare.com/ajax/libs/p5.js/0.4.12/p5.js"></script>
-<script src="https://cdn.rawgit.com/molleindustria/p5.play/master/lib/p5.play.js"></script>
-<script src="game.js"></script>
-```
+Our project's code is going to be divided into two files: `index.html` (the HTML code) and `game.js` (the JavaScript code). The HTML file will tell the browser about the game's existence and how to display it on the page. The JavaScript file will tell the browser how to actually run the game.
 
-The first line grabs the p5js library from where it's hosted on Cloudflare. The second grabs the p5play library that we'll need to build our game. And the third tells our HTML file about the external JS file where we'll write our game.
+1. Create a new directory in your Cloud9 workspace called `dodge`
+2. Create a file and copy and paste the following into it. Save it as `index.html` inside of the `dodge` directory.
 
-Create a new file named `index.html` in your directory, and throw those `<script>` tags in the `<body>`. Spin up the live preview and prepare to be blown away ... by nothing.
```

Comment:
YO I HAD A NICE NARRATIVE GOIN ON HERE :( Why did you remove my rhetorical flair